.gitignore

```diff
@@ -28,3 +28,7 @@ Session.vim
 *~
 tags
 .pem
+
+
+# Post-test cruft
+Gemfile.lock
```

Comment:
So im just running .bonnyci/run.sh to execute tests locally, and I end up with changes to the .lock file.